syslog-ng/main.c

```diff
@@ -270,6 +270,7 @@ main(int argc, char *argv[])
   app_startup();
   main_loop_init(main_loop, &main_loop_options);
   rc = main_loop_read_and_init_config(main_loop);
+  log_msg_stats_global_init();
```

Comment:
What if we could create a `register_stats_level_dependent_global_counters` method?
(and check all the global `init` functions that are registers counters... and move to there the registration?)